README.md

```diff
@@ -1,2 +1,56 @@
-# stream-it-capstone
-Javascript Capstone project 
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+<p align="center">Desktop view</p>
+
+## Preview
+![Preview](./src/assets/streamdsktphmpage.png)
+![Preview](./src/assets/st-desk-1.png)
+![Preview](./src/assets/st-desk-2.png)
+
+<h1 align="center"> Stream-It </h1>
+
```

Comment:
**You made a great effort on this project.** 
_However, some changes need to be completed to make it approvable!_

- [ ] I cannot see your presentation video about this project. Please add a link to the readme file about your presentation.

> HINTS: 
The student recorded a video presenting the features of the project.
The maximum length of the video is 5 minutes.
The video contains a clear and objective description of the project in professional English. The topic and purpose of the project are made clear.
 All the members in the video should appear in the video and talk about the project.
The video contains a highlight about a piece of the code the student finds interesting or is proud of making.

_I would suggest using Zoom to record the video!_

Reply:
Hi @Rustamxon7  thanks for the review, I have added the video link to the README.